runtime/doc/builtin.txt

```diff
@@ -2563,22 +2563,23 @@ function({name} [, {arglist}] [, {dict}])
 			function Callback() dict
 			   echo "called for " .. self.name
 			endfunction
-			...
+			"...
 			let context = {"name": "example"}
 			let Func = function('Callback', context)
-			...
+			"...
 			call Func()	" will echo: called for example
 <		The use of function() is not needed when there are no extra
-		arguments, these two are equivalent: >
+		arguments, these two are equivalent, if `Callback()` is defined
+		as `context.Callback()`: >
```

Comment:
```suggestion
		arguments, these two are equivalent, if Callback() is defined
		as context.Callback(): >
```

Reply:
This is why we can't have nice things...